README.md

```diff
@@ -326,6 +326,24 @@ pub enum Gender {
   But it is possible to place `serde` derive tags onto the generated types, so
   the same structure can support both `prost` and `Serde`.
 
+2. **I get errors when trying to run `cargo test` on MacOS**
+
+  If the errors are about missing `autoreconf` or similar, you can probably fix
+  them by running
+
+  ```
+  brew install automake
+  brew install libtool
+  ```
+
+3. **I'm getting compile errors when compiling a 2018 edition project**
+
+  If you're using prost-build, you need to use the `build-2018` feature:
```

Comment:
WDYT about `edition-2018` as the feature flag?  I think that will be clearer when encountered out of context.

Reply:
My reasoning was to make a distinction between the source code of Prost and the code we actually generate. But I'm happy to use `edition-`